db/constants.conf

```
@@ -3992,8 +3992,48 @@ constants_db: {
 	HAT_EF_SUBJECT_AURA_RED:       84
 	HAT_EF_C_SHINING_ANGEL_WING:   85
 	HAT_EF_MAGIC_STAR_TW:          86
-	HAT_DIGITAL_SPACE:             87
+	HAT_EF_DIGITAL_SPACE:          87
```

this constant has been inside repo for over 1 year,
it should be deprecated instead of silently remove it
```suggestion
	HAT_DIGITAL_SPACE: {
		Value: 87
		Deprecated: true
	}
```